modules/godbcbackend/sodbc.cc

```diff
@@ -344,13 +344,13 @@ SSqlStatement* SODBCStatement::nextRow(row_t& row)
     d_result = SQLFetch(d_statement);
     // cerr<<"subsequent SQLFetch done, d_result="<<d_result<<endl;
     if(d_result == SQL_NO_DATA) {
-      SQLRETURN result = SQLMoreResults(d_statement);
-      // cerr<<"SQLMoreResults done, result="<<d_result<<endl;
-      if (result == SQL_NO_DATA) {
-        d_result = result;
+      SQLRETURN result2 = SQLMoreResults(d_statement);
+      // cerr<<"SQLMoreResults done, result="<<d_result2<<endl;
```

Comment:
Should either be `d_result` or `result2` (likely this one). Or perhaps we should remove this line :)